### DISCUSSION TITLE
Less Intrusive Sync Notifications

### DISCUSSION THREAD

**Discussion by samawelch** [remotely-save/remotely-save#496] [Q&A] [answered]
Hi all,
Thanks for the great plugin. After a bit of fumbling around I've been able to get files to sync between my phone and PC using OneDrive. However, I'm finding the notifications (or whatever the right word is) a little annoying, as with Sync on Save set to the recommended 10 seconds, if I create several notes in a short period of time I'll get multiple sync notifications.
![image](https://github.com/remotely-save/remotely-save/assets/36561750/7203b01b-a6c8-428f-b5ea-d4974eb0cc4d)
The information is useful for debugging, but as things stand it's not actionable and blocks UI elements. I also find it pretty distracting.

As I'm pretty new to Obsidian there might be a super-obvious way to turn this off, but I can't find it. Does anyone have any suggestions to reduce or, ideally, remove these messages entirely? Thanks!

**Comment by fyears** [comment, marked as answer]
please update to the latest version 0.3.40 which fixed this

**Reply by samawelch** [reply]
That's perfect, thank you.

**Comment by samawelch** [comment]
@fyears sorry to re-open this, but although the issue seemed to go away for a while, I still get a bunch of arguably extraneous messages. Alter Console Log Level is set to info, but is there a way to set it to none and just reply on the sync messages on the bottom bar?
